pulsar-io/netty/src/main/java/org/apache/pulsar/io/netty/server/NettyServer.java

```diff
@@ -96,7 +96,7 @@ private void runUdp() throws InterruptedException {
         Bootstrap bootstrap = new Bootstrap();
         bootstrap.group(workerGroup);
         bootstrap.channel(NioDatagramChannel.class);
-        bootstrap.handler(new NettyChannelInitializer(new NettyServerHandler(this.nettySource)))
+        bootstrap.handler(new NettyChannelInitializer(new NettyUDPServerHandler(this.nettySource)))
```

Comment:
This change lgtm. 
Just have a question maybe not related with this issue. Since this is replaced with `NettyUDPServerHandler`, Is there a need to replace the `NettyServerHandler` in function `runTcp` below in line 108?

Reply:
No, the NettyUDPServerHandler is specific to UDP to manage DatagramPacket instead of byte[] payloads.

Reply:
no need for a NettyTCPServerHandler for runTcp?

Reply:
Yes, we can rename NettyServerHandler into NettyTCPServerHandler as NettyServerHandler is already the TCP handler